components/segmenter/tests/testdata/GraphemeBreakExtraTest.txt

```diff
@@ -6,110 +6,3 @@
 ÷ 0915 × 094D × 0308 ÷ 0600 ÷ # ÷ DEVANAGARI LETTER KA (ConjunctLinkingScripts_LinkingConsonant) × DEVANAGARI SIGN VIRAMA (Extend_ConjunctLinkingScripts_ConjunctLinker_ExtCccZwj) × COMBINING DIAERESIS (Extend_ExtCccZwj) ÷ ARABIC NUMBER SIGN (Prepend) ÷
 ÷ 0915 × 094D × 0308 ÷ 1F1E6 ÷ # ÷ DEVANAGARI LETTER KA (ConjunctLinkingScripts_LinkingConsonant) × DEVANAGARI SIGN VIRAMA (Extend_ConjunctLinkingScripts_ConjunctLinker_ExtCccZwj) × COMBINING DIAERESIS (Extend_ExtCccZwj) ÷ REGIONAL INDICATOR SYMBOL LETTER A (RI) ÷
 ÷ 0915 × 094D × 0308 ÷ 1F476 ÷ # ÷ DEVANAGARI LETTER KA (ConjunctLinkingScripts_LinkingConsonant) × DEVANAGARI SIGN VIRAMA (Extend_ConjunctLinkingScripts_ConjunctLinker_ExtCccZwj) × COMBINING DIAERESIS (Extend_ExtCccZwj) ÷ BABY (ExtPict) ÷
-# The following 100 monkey tests were generated from
```

Comment:
Question: Are these tests no longer correct under version 16.0? Or some random tests are regenerated in GraphemeBreakRandomTest.txt?

Reply:
I moved it to GraphemeBreakRandomTest.txt.